api/views/andela_centres.py

```diff
@@ -36,12 +38,48 @@ class AndelaCentreViewset(ModelViewSet):
     permission_classes = [IsAuthenticated, IsAdminUser]
```

Comment:
We could add 2 permission classes 
- `IsAdminReadOnly` - which allows read-only access for normal admins and 
- `IsSuperAdmin` which allows access to super admins.
this way they can be used elsewhere if needed, and leave the create method to its default

Reply:
@eric this sounds nice